libs/core/src/types.ts

```diff
@@ -3,6 +3,7 @@ export interface TrueAffectedProject {
   sourceRoot: string;
   tsConfig?: string;
   implicitDependencies?: string[];
+  targets: string[];
```

Comment:
Need to add this to the `turbo` `getTurboTrueAffectedProjects` return, not sure the structure of the project file though. Does turbo have a way to get the targets for a project?

Reply:
Made it optional for the time being so we do not need to implement it for turbo right away unless its an easy fix, I just don't know much about turbo